Scheduler/tests/query_and_cancel_test.py

```diff
@@ -24,7 +24,7 @@
 LOGGER.addHandler(logging.StreamHandler(sys.stdout))
 
 # test parameters
-TEST_ECR_URI = "754106851545.dkr.ecr.us-west-2.amazonaws.com/pr-tensorflow-training:2.2.0-gpu-py37-cu101-ubuntu18.04-example-pr-269-2020-06-11-22-13-27"
+TEST_ECR_URI = "763104351884.dkr.ecr.us-east-1.amazonaws.com/tensorflow-training:2.2.0-gpu-py37-cu101-ubuntu18.04"
```

Comment:
Tests run in us-west-2 will fail if you use the image from us-east-1. Set the region to be "us-west-2".